src/peft/peft_model.py

```diff
@@ -337,6 +338,32 @@ def from_pretrained(
         else:
             raise ValueError(f"The input config must be a PeftConfig, got {config.__class__}")
 
+        weight_map = dict(named_module_tensors(model, recurse=True))
+
+        disk_modules = []
+        index = None
+        for name, module in model.named_modules():
+            if hasattr(module, "_hf_hook") and hasattr(module._hf_hook, "original_devices"):
+                index = module._hf_hook.weights_map.dataset.index
+                for key in module._hf_hook.original_devices.keys():
+                    if dict(module._hf_hook.original_devices)[key] == torch.device("meta"):
+                        disk_modules.append(str(name) + "." + str(key))
+
+        if disk_modules and hasattr(kwargs, "use_safetensors") and not kwargs["use_safetensors"]:
+            raise AssertionError("Disk offloading not currently only supported for safetensors")
```

Comment:
```suggestion
            raise ValueError("Disk offloading currently only supported for safetensors")
```

That's what you mean, right?